repos/spack_repo/builtin/packages/openblas/package.py

```diff
@@ -162,6 +162,9 @@ class Openblas(CMakePackage, MakefilePackage):
     # https://github.com/OpenMathLib/OpenBLAS/pull/1703
     patch("openblas-0.3.2-cmake.patch", when="@0.3.1:0.3.2")
 
+    # https://github.com/OpenMathLib/OpenBLAS/issues/5473
+    patch("openblas-0.3.30-apple-LTO.patch", when="@0.3.30 platform=darwin")
```

Comment:
```suggestion
    # https://github.com/OpenMathLib/OpenBLAS/issues/5473
    patch(
        "https://github.com/OpenMathLib/OpenBLAS/commit/c1f607c43c5a1f3501cf69f0ece7e8f0a7da3e65.patch?full_index=1,
        sha256="e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b85",
        when="@0.3.30 platform=darwin",
    )
```

@Chrismarsh instead of adding the patch here and increasing the size of the `spack-packages` repository it's recommended to instead point to the upstream merge commit when possible.

Reply:
I agree. The problem is that the patch applies on top of tag 0.3.30. However the pr was rebased and merged into develop. I tried and the patch derived on develop doesn't apply to tag 0.3.30. So unless there is something I'm missing I think the standalone patch file is the best option?

Reply:
Ahh darn, I agree in this case the best option is likely to include the modified patch in the repository. 😢